src/_posts/2020-02-20-remote-pairing-a-true-story.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+asset-type: post
+name: remote-pairing-a-true-story
+title: Remote Pairing a true story
+date: 2020-02-20 00:00:00 +00:00
+author: Gonzalo Gomez Sullain
+description: Describe the ways to approachach pair programming with distributed teams from a personal experience
+image:
+   src: /assets/custom/img/blog/2020-02-20-remote-pairing-a-true-story/post-main.jpg
+   attribution:
+      text: Photo by Michael Hoyt on Unsplash
+      href: https://unsplash.com/photos/K6PQngnMdDE
+tags:
+    - software craftsmanship
+    - life at codurance
+    - pair programming
+---
+
+## Context
+
+When you join a project, you might face a brand new world. New stakeholders to deal with, new technological challenges and probably new team members which whom you might have never worked before. On the other hand, new expectations and white paper to start from the scratch defining the practices the team is going to embrace based on each of the members background.
+
+The challenge goes beyond those inherent to the project: is key to establish the new team dynamics, how the work is splitted, how requirements are refined and the workload is handled.
+
+There are several pilars to build a team uppon, if you want to levarage the chances your poject to suceed, undoubtely embracing [XP Practies](http://www.extremeprogramming.org/) should be in your table, and engaging any agile flavour to handle the business delivery workload is for sure in the handbook of a good project coordinator.
+
+## The Challenge
+Now imagine the team is composed of skilled and experienced members, but they are in different locations, or perhaps working remote, or working alternate days from home, or even in different time zone.
+
+This is a perfectly valid team set up in our brand new highly connected world, and defy the traditional ways of working, and put our engineering practices under the loupe, make us thrive to find the best to coupe with the new context, but still using the best known practices in its escense.
```

Comment:
should be essence instead of escense I think

Reply:
I think I need a EN linter in Sublime 🤕